tests/test_api/test_endpoints.py

```diff
@@ -270,12 +270,12 @@ def test_submit_changesheet():
     assert True
 
 
-@pytest.mark.skip(
-    reason="Skipping because race condition causes  http://fastapi:8000/nmdcschema/ids/nmdc:wfrqc-11-t0tvnp52.2 to 404?"
-)
+#@pytest.mark.skip(
+#    reason="Skipping because race condition causes  http://fastapi:8000/nmdcschema/ids/nmdc:wfrqc-11-t0tvnp52.2 to 404?"
+#)
```

Comment:
Delete these lines. They are recoverable via git history.

Reply:
Thanks, @dwinston. I'll do that here now @aclum (via `github.dev`)